### PR TITLE
J52: - streaming & service menu, * added a link for podcasts

### PR DESCRIPTION
**Pull request by EugeniusGR**
VIEW:
image

### CODE REVIEW

**Comment by EugeniusGR** [comment]
[Clean app menu](https://trello.com/c/Jl0v1Avu/52-clean-app-menu)

File: frontend/src/components/common/header/header.tsx [hunk 1]
```diff
@@ -23,9 +23,9 @@ const Header: React.FC = () => {
         {hasUser ? (
           <>
             <ul className={styles.navigation}>
-              <li className={styles.liNavigation}>Podcasts</li>
-              <li className={styles.liNavigation}>Streaming</li>
-              <li className={styles.liNavigation}>Setting</li>
+              <Link to={AppRoute.ROOT}>
+                <li className={styles.liNavigation}>Podcasts</li>
+              </Link>
```
Comment (ghost) [inline frontend/src/components/common/header/header.tsx]:
```suggestion
             
                <li className={styles.liNavigation}>
                 <Link to={AppRoute.ROOT}>
                   Podcasts
                  /Link>
                 </li>
              
```
`ul` element can include only `li` elements

**Review by ghost — CHANGES_REQUESTED** [review]

File: frontend/src/components/common/header/header.tsx [hunk 2]
```diff
@@ -23,9 +23,11 @@ const Header: React.FC = () => {
...
           <>
             <ul className={styles.navigation}>
-              <li className={styles.liNavigation}>Podcasts</li>
-              <li className={styles.liNavigation}>Streaming</li>
-              <li className={styles.liNavigation}>Setting</li>
+              <li>
+                <Link to={AppRoute.ROOT} className={styles.liNavigation}>
+                  Podcasts
+                </Link>
+              </li>
```
Comment (ghost) [inline frontend/src/components/common/header/header.tsx]:
```suggestion
              <li className={styles.liNavigation}>
                <Link to={AppRoute.ROOT} className={styles.link}>
                  Podcasts
                </Link>
              </li>
```

File: frontend/src/components/common/header/styles.module.scss [hunk 3]
```diff
@@ -29,6 +29,7 @@
 .liNavigation {
   margin-right: 50px;
   color: #ffffff;
+  text-decoration: none;
```
Comment (ghost) [inline frontend/src/components/common/header/styles.module.scss:35]:
```suggestion
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]
👍

**Pull request merged**